### ISSUE TITLE
508-defect-1❗️ Launchblocker [COGNITION, SCREEN READER]: Inputs MUST have unique and accessible labels

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#26701]
## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _First name only. Delete if not applicable._

## Details

When more than one area of disagreement entry is edited on the review & submit page _and_ has more than one entry with "Something else" selected, the associated inputs do not include a unique ID. Three axe violations appear:

1. IDs used in ARIA and labels must be unique
2. id attribute value must be unique
3. Form field must not have multiple label elements

## Acceptance Criteria

- [ ]  All instances of the error are fixed
- [ ]  Axe-browser plugin or axe-core CLI scans show 0 Section 508, WCAG2A or WCAG2AA violations

## Steps to Recreate

1. Go to https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/introduction
2. Log in to one of the HLR test users - https://dsva.slack.com/archives/C5AGLBNRK/p1601069669025200?thread_ts=1601069567.024300&cid=C5AGLBNRK
3. Navigate through the form, selecting at least 2 issues (eligible or addition issues)
4. On the pages asking "Tell us why you disagree with our decision", select "Something else" and enter any text.
5. Once on the review & submit page open the "Issues for review" accordion
5. Edit 2 or more issues (above the "Opt in" section).
6. Run axe DevTools and note the Id and form label errors similar to the screenshot 

## Proposed Solution (if known)

Will require a change to platform code to include an index to the array form elements

## Screenshots or Trace Logs

![Screen Shot 2021-06-28 at 3 25 30 PM](https://user-images.githubusercontent.com/136959/123700789-8b28dd00-d826-11eb-8265-78c51eba772c.png)

**Comment by Mottie** [comment]
@joshkimux This fix is now available on staging. Please validate at your convenience.

**Comment by joshkimux** [comment]
Validated! 👏
